app/javascript/api/ComparisonAPI.jsx

```diff
@@ -1,9 +1,9 @@
 import axios from 'axios';
```

Comment:
The `ComparisonAPI` class has been modified to accept multiple slugs. Ensure that the `slugs` parameter is properly sanitized before being used in the API request to prevent injection attacks.